configs/launch-os.json

```diff
@@ -4,7 +4,7 @@
          "region": "iad",
          "type": "t3.small",
          "ondemand": 0,
-         "spot": 18,
+         "spot": 72,
```

Comment:
you need 100 nodes.  4 shards, 50 nodes each, but 25 of them are our nodes. so you need 100 -4 = 96 nodes here.

Reply:
right, 4 shards X 24 = 96 nodes. I multipled by 3 shrads.